### ISSUE TITLE
I would like to include the Favicon for Google to pickup

### ISSUE THREAD

**Issue by designsonline** [Auctollo/google-sitemap-generator#107]
There doesnt seem to be any option to include the favicon anywhere on the sitemap?

**Comment by ftownes** [comment]
@designsonline you mean that you wish to have your favicon URL in the sitemap file(s)?

**Comment by designsonline** [comment]
Yes the website has a favicon, (correct name, correct size, plus specific variations for the different browsers) but it just hasn’t been found by Google yet

 

From: Frederick Townes ***@***.***>
Reply-To: Auctollo/google-sitemap-generator ***@***.***>
Date: Wednesday, 19 July 2023 at 15:36
To: Auctollo/google-sitemap-generator ***@***.***>
Cc: ***@***.***" ***@***.***>, Mention ***@***.***>
Subject: Re: [Auctollo/google-sitemap-generator] I would like to include the Favicon for Google to pickup (Issue #107)

 

GitHub, is not for support, but I will ask, have you already tried this?

**Comment by ftownes** [comment]
Alright, do you have an article that shows that sitemaps can be used to solve this problem?
